### PR TITLE
💄(llm): ui and message on receive confirmation screen

### PR DESCRIPTION
**Pull request by LucasWerey**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - little UI rework of receive screen
  - add of warning text for utxos addresses

### 📝 Description

Rework of the UI of the screen to avoid weird scroll and change animation to use reanimated 
Add message for utxos accounts

UI : 

https://github.com/user-attachments/assets/e860cdf5-5b84-4421-9123-8bce8c83e93c

Message : 

https://github.com/user-attachments/assets/5ec9d117-b1db-43b7-b5de-3f208db11d7e

### ❓ Context

- **JIRA or GitHub link**: [LIVE-14055](https://ledgerhq.atlassian.net/browse/LIVE-14055)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-14055]: https://ledgerhq.atlassian.net/browse/LIVE-14055?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.tsx [hunk 1]
```diff
@@ -255,8 +276,24 @@ function ReceiveConfirmationInner({ navigation, route, account, parentAccount }:
...
+    "litecoin",
+    "zcash",
+    "cardano",
+  ];
+  const isNotATokenAccount = (account: TokenAccount | Account): account is Account => {
+    return (account as Account).currency !== undefined;
+  };
+
+  const isAnAccount = isNotATokenAccount(account);
+  const isUTXOCompliant = isAnAccount && UTXOFamily.includes(account.currency.id);
```
[the remark this answers is not in the document]
Comment (LucasWerey) [reply apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.tsx]: Done ✅

File: apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.tsx [hunk 2]
```diff
@@ -255,8 +276,24 @@ function ReceiveConfirmationInner({ navigation, route, account, parentAccount }:
         : null;
   }
 
+  const UTXOFamily = [
```
[the remark this answers is not in the document]
Comment (LucasWerey) [reply apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.tsx]: done ✅

File: apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.tsx [hunk 3]
```diff
@@ -255,8 +277,15 @@ function ReceiveConfirmationInner({ navigation, route, account, parentAccount }:
         : null;
   }
 
+  const isNotATokenAccount = (account: TokenAccount | Account): account is Account => {
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.tsx:280]: You can do just `account.type === "Account"` imo

File: libs/live-nft-react/src/hooks/useUtxoCompliant.ts
Comment (mcayuelas-ledger) [inline libs/live-nft-react/src/hooks/useUtxoCompliant.ts:1]: It's not a hook :)

File: apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.tsx (hunk 3 above)
Comment (LucasWerey) [reply apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.tsx]: Indeed ^^

File: libs/live-nft-react/src/index.ts [hunk 4]
```diff
@@ -5,4 +5,5 @@ export * from "./hooks/useNftFloorPrice";
 export * from "./hooks/useRefreshMetadata";
 export * from "./hooks/useCheckSpamScore";
 export * from "./hooks/useFetchOrdinals";
+export * from "./hooks/helpers/utxoCompliant";
```
Comment (mcayuelas-ledger) [inline libs/live-nft-react/src/index.ts:8]: it's not well structured here, 

`helpers` should be outside `hooks` imo but it's ok :)

**Review by mcayuelas-ledger — APPROVED** [review]

File: libs/live-nft-react/src/index.ts (hunk 4 above)
Comment (LucasWerey) [reply libs/live-nft-react/src/index.ts]: moved to common/currencies/helper finally

**Review by cgrellard-ledger — APPROVED** [review]

**Pull request merged**